FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -130,6 +149,47 @@ const myAuctionConfig = {
 Note that since the `serverResponse` field in the config is a promise it is
 possible to run the on-device auction in parallel with the B&A auction.
 
+## Request Size and Configuration
+
+### Request Size Controls
+The `requestSize` field provided to `navigator.getInterestGroupAdAuctionData()`
+can be used to specify the maximum size of the returned request. If the
+`perBuyerConfig` field is not present (or is empty) then the `requestSize` field
+sets the maximum size for the returned encrypted blob. If the blob fits into a
+size bucket smaller than `requestSize` then that size will be used instead.
```

Comment:
Isn't this only true when there is no `perBuyerConfig`?  If so, we should mention it here, otherwise the next sentence contradicts it.

Reply:
Removed the redundant sentence and moved the "If the..." to be with the correct sentence.